README.md

```diff
@@ -1,24 +1,107 @@
-# README
+# Hello Rails From React
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Description the project.
 
-Things you may want to cover:
+This is a simple web app to test working with rails and react
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Languages:
+  _**Ruby**_
+  _**React**_
+  _**Redux**_
+  _**Webpack**_
+  _**Postgresql**_
+- Frameworks: _**Ruby On Rails**_
+- Technologies used: _**Visual Studio Code**_
 
```

Comment:
- [ ] Good job so far 👏 your README doesn't look professional yet but it's close! Please use the new update [README template](https://github.com/microverseinc/readme-template) provided by Microverse. Remember to add content in each section in the template 😉 By doing this, It will look more professional 😃

Reply:
Thanks a lot, @BertrandConxy for pointing out these errors🙌, This was a mistake on my end and  I will fix them immediately.